packages/client/hmi-client/src/components/models/tera-stratified-value-matrix.vue

```diff
@@ -159,27 +156,28 @@ function updateModelConfigValue(variableName: string) {
 function configureMatrix() {
 	console.log(props.modelConfiguration.configuration);
 
+	const result = getAMRPresentationData(props.modelConfiguration.configuration);
+
 	// Get only the states/transitions that are mapped to the base model
 	const matrixData =
 		props.nodeType === NodeType.State
-			? getCatlabStatesMatrixData(props.modelConfiguration.configuration).filter(
-					(d) => d['@base'] === props.id
-			  )
-			: getCatlabTransitionsMatrixData(props.modelConfiguration.configuration).filter(
-					(d) => d['@base'] === props.id
-			  );
-
-	console.log(matrixData);
+			? result.stateMatrixData.filter((d) => d._base === props.id)
+			: result.transitionMatrixData.filter((d) => d._base === props.id);
 
 	if (isEmpty(matrixData)) return;
 
 	// Grab dimension names from the first matrix row
 	const dimensions = [cloneDeep(matrixData)[0]].map((d) => {
-		delete d.id;
-		delete d['@base'];
+		delete d._id;
+		delete d._base;
 		return Object.keys(d);
 	})[0];
 
+	// FIXME: resolve id vs _id
+	matrixData.forEach((d) => {
+		d.id = d._id;
+	});
```

Comment:
I will resolve the _id vs id in a subsequent PR,  as the scope is already larger than what I would've liked.